Aztec/Classes/TextKit/TextView.swift

```diff
@@ -972,18 +972,28 @@ open class TextView: UITextView {
         return storage.attachment(withId: id)
     }
 
-    /// Removes the attachments that match the attachament identifier provided from the storage
+    /// Removes the attachment that match the attachment identifier provided from the storage
     ///
     /// - Parameter attachmentID: the unique id of the attachment
     ///
     open func remove(attachmentID: String) {
-        storage.remove(attachmentID: attachmentID)
+        guard let range = storage.rangeFor(attachmentID: attachmentID) else {
+            return
+        }
+        let originalText = storage.attributedSubstring(from: range)
```

Comment:
From line 983 to line 990 we're repeating code that's in several other places in `TextView.swift`

Why not implement in `TextView.swift` method

```
func replaceCharacters(in: range, with attrString: NSAttributedString)
```

 to take care of unified undo support?